engineyard/ey-cookbooks-stable-v5
# Rename cron to ey-cron

## Description of your patch

Rename cron -> ey-cron. Refactoring needed to be able to port the custom cron cookbook from v4.

## Recommended Release Notes

None.

## Estimated risk

High. Need to make sure customer's cron jobs disappear.

## Components involved

ey-cron main Chef cookbook

## Description of testing done

While on the root directory of the main chef directories, upload to the target environment. Applying the recipes will override cookbooks that refers to the cron cookbook.


## QA Instructions
1. Boot an environment (solo and/or cluster)
2. Ensure chef runs to completion successfully and app deploys and runs as expected
3. Create a cron job from the dashboard using both the root and deploy user.
4. Apply changes.
5. 
5. Upgrade to target QA stack.
7. Click Apply.
8. Verify again that the cronjobs are inside the solo and app_master instance
9. Terminate the environment
10. Boot the environment again
11. Verify again that the cronjobs are inside the solo and app_master instance
